chat-widget/src/components/livechatwidget/common/createMarkdown.ts

```diff
@@ -16,8 +17,7 @@ export const createMarkdown = (disableMarkdownMessageFormatting: boolean, disabl
                 breaks: (!disableNewLineMarkdownSupport)
             }
         );
-        // ToDo: Commenting below usage of plugin until deferred bug is resolved: https://github.com/mayashavin/markdown-it-slack/issues/1
-        // markdown.use(MarkdownSlack);
+        markdown.use(MarkdownSlack);
```

Comment:
so.. there was no need to update the version ?,  the bug mentions the intention to build a new release version for slack-markdown....

Reply:
I had the same doubt. Maybe the same version was patched? Either way the original bug doesn't occur anymore